### PR TITLE
Finished Machining App 

### PR DESCRIPTION
**Pull request by blueskye04**

### CODE REVIEW

**Review by blueskye04 — COMMENTED** [review]
done

File: Front-End/src/PageM.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ function BumpersDataTable(props) {
 
                 <p className="selection-label" id="Bumpers-Data-Table">Bumpers Data Tables</p>
                 <div className="textArea">
-                    <table border="1" width="100%">
+                    <table border="1" width="200%">
```
Comment (NathanDuPont) [inline Front-End/src/PageM.js:145]: With this set to `200%`, it ends up putting the table way off the side of the screen. Is this what you wanted, or did you want to fit it all on one screen? 

image

**Comment by blueskye04** [comment]
fixed.

File: Front-End/src/PageM.js (hunk 1 above)
Comment (NathanDuPont) [reply Front-End/src/PageM.js:172]: going to fix with future css

**Review by NathanDuPont — APPROVED** [review]

**Review by NathanDuPont — APPROVED** [review]

**Pull request merged**